test/parallel/test-node-output-errors.mjs

```diff
@@ -43,6 +47,16 @@ describe('errors output', { concurrency: true }, () => {
     { name: 'errors/throw_in_line_with_tabs.js', transform: errTransform },
     { name: 'errors/throw_non_error.js', transform: errTransform },
     { name: 'errors/promise_always_throw_unhandled.js', transform: promiseTransform },
+    { name: 'errors/if-error-has-good-stack.js', transform: errTransform },
+    { name: 'errors/test-no-extra-info-on-fatal-exception.js', transform: errTransform },
+    { name: 'errors/throw_error_with_getter_throw.js', transform: errTransform },
+    { name: 'errors/throw_null.js', transform: errTransform },
+    { name: 'errors/throw_undefined.js', transform: errTransform },
+    { name: 'errors/timeout_throw.js', transform: errTransform },
+    { name: 'errors/undefined_reference_in_new_context.js', transform: errTransform },
+    { name: 'errors/util_inspect_error.js', transform: customTransform },
+    { name: 'errors/util-inspect-error-cause.js', transform: customTransform },
```

Comment:
@MoLow 

This test case is the one causing the headaches.

See: https://github.com/nodejs/node/pull/47868#issuecomment-1536937028

Reply:
Resolved by using { tty: true } and using a custom normalization